minethd.cpp

```diff
@@ -530,3 +560,446 @@ void minethd::double_work_main()
 	cryptonight_free_ctx(ctx0);
 	cryptonight_free_ctx(ctx1);
 }
+
```

Comment:
**Mistake!**
Probably (AStyle) Styling error/incosistiency leading to false +ve changes.
Nothing to do with the code.

Sorry!